apps/src/lib/kits/maker/dropletConfig.js

```diff
@@ -304,14 +302,14 @@ const circuitPlaygroundBlocks = [
     func: 'buzzer.playNotes',
     category: CIRCUIT_CATEGORY,
     paletteParams: ['notes', 'tempo'],
-    params: [stringifySong(SONG_1D), 120],
+    params: [stringifySong(SONG_SINGLE_NOTE), 120],
     paramButtons: {minArgs: 1, maxArgs: 2}
   },
   {
     func: 'buzzer.playSong',
     category: CIRCUIT_CATEGORY,
     paletteParams: ['notes', 'tempo'],
-    params: [stringifySong(SONG_CHARGE), 120],
+    params: [`[${stringifySong(SONG_CHARGE[0])}]`, 120],
```

Comment:
nit: `SONG_CHARGE[0]` is the same as `SONG_SINGLE_NOTE` -- should we use that instead?

Reply:
SONG_CHARGE[0] is ['G3', 0.25] and SONG_SINGLE_NOTE is ['G3']. PlayNotes takes a one dimensional value (note) and PlaySong takes two dimensional (note, length).

Reply:
ahh, my bad!